code/game/gamemodes/changeling/changeling.dm

```diff
@@ -110,6 +110,17 @@ GLOBAL_VAR(changeling_team_objective_type) //If this is not null, we hand our th
 	user.update_body()
 	user.domutcheck()
 
+	// Do skillchip code after DNA code.
+	// There's a mutation that increases chip capacity, even though we force-implant skillchips.
+
+	// Remove existing skillchips.
+	user.destroy_all_skillchips(silent = FALSE)
+
+	// Add new set of skillchips.
+	for(var/S in chosen_prof.skillchips)
+		var/obj/item/skillchip/skillchip = new S(src, FALSE)
```

Comment:
```suggestion
	for(var/skill_chip in chosen_prof.skillchips)
		var/obj/item/skillchip/skillchip = new skill_chip(src, FALSE)
```
* Better variable names. Come on, I've done this same review several times already on this PR.